conf/application.conf

```diff
@@ -145,13 +145,20 @@ microservice {
             authorization-token-1404 = "localIFToken1404"
             environment = "localIFEnvironment"
         }
+
+        hip {
+            url = "http://localhost:9084"
```

Comment:
know other apis urls are already like this but wondering if it's worth splitting out the url into host and port? and we build the url up in code instead?

Reply:
Yeah that make sense but not sure how it will generate url with port number as actual HIP ones do not have one in API Spec.
Few HIP platform APIs migrated have this port number passed in as well but they are not live.
So considering unknowns I left it as is for now, but could do it as improvement in next release(R16) when we have sometime, if that's ok please.

Reply:
Though 443 is default port, it shouldn't cause issues. I'll make them

Reply:
<img width="389" alt="Screenshot 2025-04-07 at 10 43 30" src="https://github.com/user-attachments/assets/4181985e-09d5-419b-bce8-75a508850255" />